docs/source/library-user-guide/using-the-sql-api.md

```diff
@@ -29,16 +29,15 @@ using the [`SessionContext::sql`] method. For lower level control such as
 preventing DDL, you can use [`SessionContext::sql_with_options`] or the
 [`SessionState`] APIs
 
-[`sessioncontext`]: https://docs.rs/datafusion/latest/datafusion/execution/context/struct.SessionContext.html
```

Comment:
I moved this down to the bottom of the doc so the links are't duplicated and I could reuse `SessionContext::sql` reference